tensorflow/compiler/mlir/tensorflow/tests/tf_interface_optimize_pass.mlir

```diff
@@ -0,0 +1,136 @@
+// RUN: tf-opt %s -tf-interface-optimize-pass | FileCheck %s
+
+// CHECK-LABEL: testSingleConj
+func @testSingleConj(%arg0: tensor<8x16x32x64xcomplex<f32>>) -> tensor<8x16x32x64xcomplex<f32>> {
+  %0 = "tf.Conj"(%arg0) : (tensor<8x16x32x64xcomplex<f32>>) -> tensor<8x16x32x64xcomplex<f32>>
```

Comment:
Can values be captured with regex and used instead of relying on the printer printing a certain value?
For this case, it can be something like

```
// CHECK-LABEL: testSingleConj
// CHECK-SAME:  ([[ARG0:%.+]]: tensor<8x16x32x64xcomplex<f32>>)
func @testSingleConj(%arg0: tensor<8x16x32x64xcomplex<f32>>) -> tensor<8x16x32x64xcomplex<f32>> {
  // CHECK: [[CONJ:%.+]] = "tf.Conj"([[ARG0]])
  %0 = "tf.Conj"(%arg0) : (tensor<8x16x32x64xcomplex<f32>>) -> tensor<8x16x32x64xcomplex<f32>>
  // CHECK: return [[CONJ]]
  return %0: tensor<8x16x32x64xcomplex<f32>>
}
```

Same with other tests below.

Reply:
https://llvm.org/docs/CommandGuide/FileCheck.html#filecheck-regex-matching-syntax has some documentation on FileCheck string regex and substitution.